baybe/constraints/continuous.py

```diff
@@ -138,6 +140,26 @@ class ContinuousCardinalityConstraint(
 ):
     """Class for continuous cardinality constraints."""
 
+    @property
+    def n_inactive_parameter_combinations(self) -> int:
+        """The number of possible inactive parameter combinations."""
```

Comment:
Can add `possible`, making it to 'n_possible_inactive_parameter_combinations'. @AdrianSosic @AVHopp your opinions?

Reply:
Tbh, I think this is very redundant and also inconsistent with our code elsewhere. When there are `N` objects available of some type, it pretty much always implies that these are also `N` *possible* objects to consider. Picking just one random example from our code: We also use `n_candidates` and not `n_possible_candidates` because it is clear that all of them are "possible choices"

Reply:
without it can be read like "the current number of inactive parameters"

Reply:
True, but:
* this is meaningless in the given context, i.e. it's part of a stateless constraint definition class
* the "combinations" also sort of indicates that it's not the "current"

I'm not saying we can't find a better name, but `n_possible_inactive_parameter_combinations` really seems a bit of an overkill. But if you all prefer it, I'll not block

Reply:
ok fine, see also that its annoyingly long then